workflow-search/values.yaml

```diff
@@ -0,0 +1,57 @@
+# Default values for workflow-search.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: icgcargo/workflow-search
+  tag: latest
+  pullPolicy: Always
+
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  paths: []
+  hosts:
+    - host: chart-example.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+appConfig:
+  PORT: 8082
+  ELASTIC_HOST: ""
+  ELASTIC_PORT: ""
+  ELASTIC_WORKFLOWINDEX: ""
+  ELASTIC_TASKINDEX: ""
+
+env: {}
```

Comment:
the deployment is not reading this object .. so the chart users doesn't have a way to override your application configurations

Reply:
https://github.com/icgc-argo/workflow-infra/pull/6

Reply:
what do you mean @blabadi , I see the env parser in deployment.yml